lib/options.js

```diff
@@ -11,12 +11,11 @@ module.exports = function parseOptions (params) {
   else if (Array.isArray(params.plugins)) plugins = params.plugins
   else plugins = params.plugins
 
-  const options = {}
-
-  if (typeof params !== 'undefined') {
-    options.parser = params.parser
-    options.syntax = params.syntax
-    options.stringifier = params.stringifier
+  const options = {
```

Comment:
We always have params as object, because we have `const options = loaderUtils.getOptions(this) || {}`